go.mod

```diff
@@ -43,6 +43,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/config/model v0.59.0-rc.5
 	github.com/DataDog/datadog-agent/pkg/config/remote v0.59.0-rc.5
 	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.59.0-rc.5
+	github.com/DataDog/datadog-operator/api v0.0.0-20250113195927-3d36de7daa54
```

Comment:
I don't think this is correct. This dependency should by managed by `go work` without pinning to a specific version.

Reply:
I added a new commit with the fix.

Reply:
Thanks for catching this!